configuration/jsr223.md

```diff
@@ -31,6 +31,90 @@ New APIs are planned to be implemented in the future, which will provide standar
 - [Javascript](jsr223-js.html)
 - Groovy (TODO)
 
+#### Example rules for a first impression
+
+:::: tabs
+
+::: tab JavaScript
+```js
+'use strict';
+
+scriptExtension.importPreset("RuleSupport");
+scriptExtension.importPreset("RuleSimple");
+
+var sRule = new SimpleRule() {
+    execute: function( module, input) {
+        print("This is a 'hello world!' from a Javascript rule.");
+    }
+};
+
+sRule.setTriggers([
+    TriggerBuilder.create()
+        .withId("aTimerTrigger")
+        .withTypeUID("timer.GenericCronTrigger")
+        .withConfiguration(
+            new Configuration({
+                "cronExpression": "0 * * * * ?"
+            })).build()
+    ]);
+
+automationManager.addRule(sRule);
+```
+:::
+
+
+::: tab Jython
```

Comment:
Jython should be first

Reply:
The question is why? :D
From my point groovy should be first, if we go for an alphabetical order.
I think we need to define some blueprint for this in general for the future.

Reply:
Sorry, I thought this was common knowledge, mainly due to Kai's comment about making Jython a first class citizen ([first link](https://community.openhab.org/search?context=topic&context_id=83943&q=Jython%20first%20class%20citizen&skip_context=true)). It is also the most used and the helper libraries are the most advanced. I'm familiar with both JavaScript and Python, and in comparison, JavaScript is definitely the least friendly for writing automation. The HL docs would need a lot of updates if the order needs to change! The tab should be Python vs Jython, since that is the programming language used. It will also migrated easier when Graal comes in. This also gives a reverse alpha all the way down to DSL Rules. :smile: jRuby is almost ready to be added and I've had a look into Kotlin too, so we should have examples for more languages soon!

Reply:
Is the plan for the Helper Libraries to become part of OH (minus the Community submissions perhaps) or are they to remain forever separate? I ask because the difference between these examples and especially the Python example using the decorators is pretty big. And the decorators make the Python look a lot more like Rules DSL in structure which will aide users who want to migrate.

Reply:
The plan for the core helper libraries is obsolescence... except for any functionality that cannot fit into the Scripting API. These examples are the raw API, which IMO people should be educated on. Once the Scripting API is available, everything simplifies and pulls back into OH.

Reply:
Hm, this is going in the wrong direction.
My main intention here was to add a TAB example and i thought jsr would be a good starter for this.
But i don't want to discuss this in general now.

So i should maybe remove this tab example.
There is still much unclear in how we should and will document jsr223.

Reply:
IMO it looks good and provides useful information. The actual content can be cleaned up later when the nonHL docs are moved from the helper library docs to the OH docs. The tabs are great!